.travis.yml

```diff
@@ -3,7 +3,7 @@ matrix:
   include:
     - os: linux
       env: NAME=Linux
-      if: type = pull_request
+      if: NOT branch =~ /^\d+\.\d+\.\d+.*$/
```

Comment:
Please add detailed comments here and for each of the changes below so that future generations may learn more quickly than we did how and why to do this kind of thing.

Reply:
(Also, there are lots of minor issues that I'd like for us to clean up in MakeOTF.py, but nothing specific to the changes.)

Reply:
done